### DISCUSSION TITLE
Trackman Support?

### DISCUSSION THREAD

**Discussion by jmjava** [springbok/MLM2PRO-GSPro-Connector#132] [General]
Has anyone thought about trackman support?

**Comment by wonder99** [comment]
No, no one has asked for that.

**Comment by wonder99** [comment]
Oh! I just noticed we already have a training file for Trackman, so support is probably already there!

**Reply by bouxy25** [reply]
Where is that training file I’m trying to connect trackman to gspro but I’m lost

**Comment by jmjava** [comment]
I actually see a train data in the repo.  I have a friend that owns one was
thinking maybe try it.

**Reply by bouxy25** [reply]
Did you ever get trackman to connect with gspro

**Comment by springbok** [comment]
I’d have to check as I can’t remember, I thought we did add TM4 support, give it a go and see if it works.

**Comment by jmjava** [comment]
No rush.  I will be speaking with him before the holidays not sure if he is
interested in GSPro yet

**Comment by bouxy25** [comment]
Why does the connector only ask for 6 parameters when I’m setting up the trackman 4 when trackman has like20+

**Comment by wonder99** [comment]
It's because the GSPRO interface doesn't support all those parameters.  It's documented here:
https://gsprogolf.com/GSProConnectV1.html

Only 5 parameters are actually needed, to completely describe the ball flight that results from a shot: ball speed, two for spin, and two for launch angles.  The rest are purely informational.

We do send some informational metrics to GSPRO in some cases, but are restricted to the API documentation above.

**Comment by bouxy25** [comment]
Thanks for your reply I’ll be more than happy to donate to the cause where
do I do that

David

**Comment by wonder99** [comment]
Springbok has a donate button in the app (coffee cup with a heart on it), which points here
https://ko-fi.com/springbok_dev

**Comment by brattzDIG** [comment]
Hi, will this connector work with Trackman iO? We’ve got 5 iO duos in a commercial facility. I’d love to be able to use GS Pro courses on them.

**Comment by bouxy25** [comment]
I too would love to see this connector work with trackman

**Reply by wonder99** [reply]
Feel free to try it, there is already some support for Trackman.  If it doesn't work, let us know and attach some high-quality screenshots here, showing all the possible numbers, + and - and L and R if they're used by Trackman.
